packages/deployment/app/cloudformation.template.yml

```diff
@@ -719,29 +606,6 @@ Resources:
           {{@key}}: {{{this}}}
       {{/if}}
     {{/each}}
-      {{# if ../api_backend_url}}
-          BACKEND_API_URL: {{../api_backend_url}}
-      {{else}}
-          BACKEND_API_URL:
-            Fn::Join: ["", [ "https://", {"Ref": "backendRestApi"}, ".execute-api.", {"Fn::Sub": "${AWS::Region}"}, ".amazonaws.com/{{../apiStage}}/"]]
-      {{/if}}
-      {{# if this.urs}}
```

Comment:
Are there users with custom lambdas that have `urs: true`? If so, we should leave this in